rviz_default_plugins/include/rviz_default_plugins/displays/wrench/wrench_display.hpp

```diff
@@ -93,6 +93,7 @@ private
 
   std::deque<std::shared_ptr<rviz_rendering::WrenchVisual>> visuals_;
 
+  rviz_common::properties::BoolProperty * accept_NaN_values_;
```

Comment:
let's keep variable style to lower-case.

```suggestion
  rviz_common::properties::BoolProperty * accept_nan_values_;
```